### ISSUE TITLE
SyncRedirector can support earlier versions of anki.

### ISSUE THREAD

**Issue by mingming1248** [kuklinistvan/docker-anki-sync-server#16]
This issue is about your plugin `SyncRedirector` (2124817646), not `docker-anki-sync-server`. `SyncRedirector` can support earlier versions of anki. 

My client is windows, and the server is `tsudoko/anki-sync-server` (not using your `docker-anki-sync-server`). 

First of all, I installed 2.1.15. I want to use `SyncRedirector`, but the version 2.1.15 is not supported.. 
After that, I installed 2.1.22, and I can use `SyncRedirector` . 
But I have another problem:"Your client is using unsupported sync protocol (10, supported version: 9)". This may be because 2.1.22 is too new. This may require `tsudoko/anki-sync-server` to provide support.
So I switched back to 2.1.15 and found that `SyncRedirector` could still be used. 

As for `tsudoko/anki-sync- server`, the author stated in README that 2.1.1 to 2.1.11 can be supported, and my tests show that 2.1.15 and 2.1.16 are also possible. 
So I think `SyncRedirector` can also support versions after 2.1.1.

Maybe you can make the `"min_point_version": 19` in `SyncRedirector`'s `meta.json` a little smaller. 
In this way, more versions can use it.

**Comment by kuklinistvan** [comment]
Hello,
thank you for your remarks! I'm a little busy now but going to check back soon!

**Comment by kuklinistvan** [comment]
Hello again,

- The message saying that "Your client is using unsupported sync protocol (10, supported version: 9)" is not from `SyncRedirector` indeed, but Anki itself, and, as you wrote, it conveys that the client requires a more recent API that the server is able to offer. It was my mistake - I've just set the supported version number to exactly 2.1.19. (see below)
- You're right that `SyncRedirector` could support more versions - in fact, if you look at the source of it, you can conclude that it is not black magic - it only presents a more convenient way to set up the custom sync urls.

However, I'm thinking of another problem right now.

We already had all the spare parts to get Anki custom sync server working without this Docker image and `SyncRedirector`. Therefore, my goal is not to make running private sync servers possible, as it is done already - thanks to people like @tsudoko or @dsnopek and their contributors, and contributors of other approaches to this problem.

Instead, what I wanted to achieve is to make its installation easy, portable and _reliable_. As Anki changes relatively frequently, I've decided that instead of trying to keep up with always the latest versions of the clients and the server, I'm providing you a tested set of versions - now even with mirrored downloads - that I've verified and seen working myself. I would like to build trust with you, therefore, I'm doing my best to ensure that the contents of this repository always can be used without headaches. (that does not mean of course, that these versions won't ever update - they do, but in a different pace)

Although I agree that `SyncRedirector` most probably could be used to support more versions of Anki, however, extending the range of accepted versions would mean that I'm taking the same level of responsibility for those to work.

**TL;DR As it varies how much time I can dedicate to support this project, I prefer to support less versions over more possible breakage.**

But, that does not mean that I can - or wish to - stop you from forking the repository or simply reuploading `SyncRedirector` with a broader range of supported versions as long as you comply with the license of the project ;)

As a sidenote, it is not specified in `meta.json` (that is a client side cache) but in the settings of the plugin in the repository of AnkiWeb.

I'm closing this now, but feel free to reopen if you have further suggestions or questions.
